### DISCUSSION TITLE
how to create convex mesh after convex decomposition？

### DISCUSSION THREAD

**Discussion by oneanime** [NVIDIA-Omniverse/PhysX#324] [Q&A]
How do I create a convex mesh after convex decomposition, but convex meshes are limited to 256 vertices

**Comment by vreutskyy** [comment]
Hi @oneanime 
Not sure I understand your question. That's the idea of convex decomposition - if your shape can't be approximated by a single convex mesh, because of the shape's concavity or complexity, it can be split onto several convex parts any of which doesn't exceed convex meshes' limitation of 256 vertices/faces. Or did you mean something else?

**Reply by oneanime** [reply]
Thank you for the explanation, the issue has been resolved. I really appreciate your help!
